src/CLI/actioner/sonic-cli-pfm.py

```diff
@@ -21,108 +21,25 @@
 import time
 import json
 import ast
-import openconfig_platform_client
-from rpipe_utils import pipestr
-from openconfig_platform_client.rest import ApiException
+import cli_client as cc
+import urllib3
 from scripts.render_cli import show_cli_output
 
-
-import urllib3
 urllib3.disable_warnings()
 
-blocked_fields = {'parent':0, 'used_power':0, 'allocated_power':0, 'temperature':0}
-plugins = dict()
-
-def filter_json_value(value):
-    for key,val in value.items():
-        if key in blocked_fields:
-            del value[key]
-        else:
-	    temp = key.split('_')
-	    alt_key = ''
-	    for i in temp:
-		alt_key = alt_key + i.capitalize() + ' '
-	    value[alt_key]=value.pop(key)
-
-    return value
-
-def register(func):
-    """Register sdk client method as a plug-in"""
-    plugins[func.__name__] = func
-    return func
-
-
-def call_method(name, args):
-    method = plugins[name]
-    return method(args)
-
-def generate_body(func, args):
-    body = None
-    # Get the rules of all ACL table entries.
-
-    if func.__name__ == 'get_openconfig_platform_components':
-        keypath = []
-
-    else:
-       body = {}
-
-    return keypath,body
-
-
 def run(func, args):
-    c = openconfig_platform_client.Configuration()
-    c.verify_ssl = False
-    aa = openconfig_platform_client.OpenconfigPlatformApi(api_client=openconfig_platform_client.ApiClient(configuration=c))
+    aa = cc.ApiClient()
 
-    # create a body block
-    keypath, body = generate_body(func, args)
-
-    try:
-        if body is not None:
-           api_response = getattr(aa,func.__name__)(*keypath, body=body)
-
-        else :
-           api_response = getattr(aa,func.__name__)(*keypath)
-     
-        if api_response is None:
-            print ("Success")
-        else:
-            api_response = aa.api_client.sanitize_for_serialization(api_response)
-            value =  api_response['openconfig-platform:components']['component'][0]['state']
-	    if value is None:
-                return
-	    if 'oper-status' in value:
-		temp = value['oper-status'].split(':')
-		if temp[len(temp) - 1] is not None:
-	            value['oper-status'] = temp[len(temp) - 1]
-            show_cli_output(sys.argv[2],filter_json_value(value))
-
-    except ApiException as e:
-        if e.body != "":
-            body = json.loads(e.body)
-            if "ietf-restconf:errors" in body:
-                 err = body["ietf-restconf:errors"]
-                 if "error" in err:
-                     errList = err["error"]
-
-                     errDict = {}
-                     for dict in errList:
-                         for k, v in dict.iteritems():
-                              errDict[k] = v
+    path = cc.Path('/restconf/data/openconfig-platform:components/component=%s'%args[0])
+    response = aa.get(path)
+    if response.ok():
+        show_cli_output(sys.argv[3], response.content)
+    else:
+        print "Failed"
```

Comment:
Lets not print just "Failed". Let's use the format "%Error: <message>". response.error_message() would give the reason.